subsys/net/lib/http/Kconfig

```diff
@@ -29,20 +29,6 @@ config HTTP_CLIENT
 	help
 	  HTTP client API
 
-config HTTP_CLIENT_RECV_RETRY
-	int "HTTP client receiver retry"
-	depends on HTTP_CLIENT
-	default 5
-	help
-	  The number of retries when EAGAIN happens
-
-config HTTP_CLIENT_RETRY_INTERVAL_MS
-	int "HTTP client retry interval in ms"
-	depends on HTTP_CLIENT
-	default 100
-	help
-	  HTTP client retry interval in ms
-
```

Comment:
You can squash this change to previous commit if the kconfig options are not used after all. No point introducing a change in one commit and then remove it in next.

Reply:
sure, I thought PRs are merged via GitHub's squash and merge, but this project prefers a local squash.